src/haddock/clis/cli_analyse.py

```diff
@@ -205,6 +153,14 @@ def update_paths(
     default=1,
 )
 
+ap.add_argument(
+    "--self-contained",
+    help="Should the models be accessed locally?",
```

Comment:
maybe consider adding a better description without question about this parameter